pkg/client/group_mgmt_client.go

```diff
@@ -393,7 +393,7 @@ func processAsyncResponse(client *GroupMgmtClient, body []byte) (interface{}, er
 			}
 		}
 		if len(jobId) == 0 {
```

Comment:
At line 375, errResp is constructed with no separator between the messages.  I'm curious to see how it looks with multiple messages.

Reply:
I can't tell that there is a separator in the message, but I'll take your word for it.